pyproject.toml

```diff
@@ -50,6 +49,26 @@ classifiers = [
 ]
 license = {text = "MIT"}
 
+[tool.pdm.dev-dependencies]
+lint = [
+    "autopep8~=2.0.2",
+    "black~=23.3.0",
+    "docutils~=0.20.1",
+    "flake8~=6.0.0",
+    "flake8-pyproject~=1.2.2",
+    "isort~=5.12.0",
+    "pre-commit~=3.3.1",
+    "yamllint>=1.35.1",
+    "mypy~=1.2.0",
+    "absolufy-imports~=0.3.1",
+    "types-requests~=2.31.0.6",
+    "types-redis~=4.6.0.3",
+    "types-PyYAML~=6.0.12.12",
+    "types-pyOpenSSL~=23.2.0.2",
+    "types-PyMySQL~=1.1.0.1",
```

Comment:
Some of these type stubs are unnecessary and won't apply to this package. I think only `types-requests` would be relevant here